gax-java/gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -74,36 +78,53 @@ public abstract class RetrySettings implements Serializable {
 
   /**
    * TotalTimeout has ultimate control over how long the logic should keep trying the remote call
-   * until it gives up completely. The higher the total timeout, the more retries can be attempted.
-   * The default value is {@code Duration.ZERO}.
+   * until it gives up completely. The higher the total timeout, the more retries and polls can be
+   * attempted. The default value is {@code Duration.ZERO}.
+   *
+   * <p>If there are no configurations, Retries have the default timeout value set above and LROs
+   * have a default total timeout value of {@code Duration.ofMillis(300000)} (5 minutes).
    */
   public abstract Duration getTotalTimeout();
 
   /**
-   * InitialRetryDelay controls the delay before the first retry. Subsequent retries will use this
-   * value adjusted according to the RetryDelayMultiplier. The default value is {@code
+   * InitialRetryDelay controls the delay before the first retry/ poll. Subsequent retries and polls
+   * will use this value adjusted according to the RetryDelayMultiplier. The default value is {@code
```

Comment:
Is this comment `The default value is {@code * Duration.ZERO}.` still valid with your new comment regarding default values?

Reply:
Updated and hopefully it's clearer what the default values are for retry and polling operations.

Reply:
Perfect! Now I see why we have two default numbers, one for Retry and one for LRO